### ISSUE TITLE
Add aria-hidden attribute to the 

### ISSUE THREAD

**Issue by abdonrd** [vaadin/vaadin-radio-button#154]
As the `input` has the `tabindex="-1"`:

https://github.com/vaadin/vaadin-radio-button/blob/5a875e267ced564c3410f24f53fc2274ef3f02c8/src/vaadin-radio-button-mixin.ts#L51

Shouldn't it also have `aria-hidden="true"`?

Reference: https://www.sarasoueidan.com/blog/keyboard-friendlier-article-listings/#optimizing-keyboard-navigation-using-tabindex-and-aria-hidden

**Comment by web-padawan** [comment]
Adding `aria-hidden` would presumably fail `axe` test, same as current implementation does.
We need to verify the proper fix that would work with screen readers and pass `axe` tests.

Currently, a11y improvements project is not the highest priority, but we will get back to it later.
